### PR TITLE
Enable travis

### PR DESCRIPTION
**Pull request by dbalabka**
It will address #3

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+language: php
+
+matrix:
+  fast_finish: true
+  include:
+    - php: '7.1'
+    - php: '7.2'
+    - php: '7.3'
+    - php: '7.4snapshot'
```
Comment (oleg-andreyev) [inline .travis.yml]: set allow to fail for 7.4

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+install: composer update --prefer-dist --no-interaction $COMPOSER_FLAGS
```
Comment (oleg-andreyev) [inline .travis.yml]: `$COMPOSER_FLAGS` missing

File: .travis.yml (hunk 1 above)
Comment (dbalabka) [reply .travis.yml]: Is there a technical reason for this? The code relies on some PHP 7.4 features with BC to stables versions

Comment (dbalabka) [reply .travis.yml]: Furthermore, PHP 7.4 in beta so there will not be any major changes. It is not a problem to already implement support of PHP 7.4

Comment (oleg-andreyev) [reply .travis.yml]: if you are sure that this feature won't break within all release cycle them okay
https://wiki.php.net/todo/php74#timetable

**Comment by aheart** [comment]
It would be nice to add build and coverage badges. :)

File: .travis.yml (hunk 1 above)
Comment (dbalabka) [reply .travis.yml]: Yes, I'm sure and everyone should be sure because there should not be a BC break. Only fixes might appear. 😃

**Comment by dbalabka** [comment]
I'm going to merge this. Seems all comments are addressed. Thanks, @oleg-andreyev and @aheart for collaboration!.

**Pull request merged**